### PR TITLE
Ava UI: Input Menu Redesign

### PR DESCRIPTION
**Pull request by IsaacMarovitz**
**Changes:**
- Refactor Input Menu to use Compiled Bindings where possible
- Cleanup of XAML
- Adjust UI to match rest of Ava UI
- [Fixes Pro Controller SVG](
https://cdn.knightlab.com/libs/juxtapose/latest/embed/index.html?uid=ff4ca41a-f466-11ed-b5bd-6595d9b17862)

Superceeds #3558

Screenshot 2023-05-16 at 23 46 36

### CODE REVIEW

File: src/Ryujinx.Ava/UI/Views/Input/MotionInputView.axaml.cs [hunk 1]
```diff
@@ -6,23 +6,22 @@
...
 using System.Threading.Tasks;
 
-namespace Ryujinx.Ava.UI.Windows
+namespace Ryujinx.Ava.UI.Views.Input
 {
-    public partial class MotionSettingsWindow : UserControl
+    public partial class MotionInputView : UserControl
     {
-        private readonly InputConfiguration<GamepadInputId, StickInputId> _viewmodel;
+        private MotionInputViewModel ViewModel;
```
Comment (AcK77) [inline src/Ryujinx.Ava/UI/Views/Input/MotionInputView.axaml.cs:13]:
```suggestion
        private MotionInputViewModel _viewModel;
```
Private field should start with `_`

File: src/Ryujinx.Ava/UI/Views/Input/RumbleInputView.axaml.cs [hunk 2]
```diff
@@ -6,34 +6,34 @@
...
 using System.Threading.Tasks;
 
-namespace Ryujinx.Ava.UI.Windows
+namespace Ryujinx.Ava.UI.Views.Input
 {
-    public partial class RumbleSettingsWindow : UserControl
+    public partial class RumbleInputView : UserControl
     {
-        private readonly InputConfiguration<GamepadInputId, StickInputId> _viewmodel;
+        private RumbleInputViewModel ViewModel;
```
Comment (AcK77) [inline src/Ryujinx.Ava/UI/Views/Input/RumbleInputView.axaml.cs:13]:
```suggestion
        private RumbleInputViewModel _viewModel;
```
Private field should start with `_`

File: src/Ryujinx.Ava/UI/Views/Input/RumbleInputView.axaml.cs [hunk 3]
```diff
@@ -6,34 +6,34 @@
...
             InitializeComponent();
-            DataContext = _viewmodel;
+            DataContext = ViewModel;
         }
 
-        public static async Task Show(ControllerSettingsViewModel viewmodel)
+        public static async Task Show(ControllerInputViewModel viewmodel)
         {
-            RumbleSettingsWindow content = new RumbleSettingsWindow(viewmodel);
+            RumbleInputView content = new RumbleInputView(viewmodel);
```
Comment (AcK77) [inline src/Ryujinx.Ava/UI/Views/Input/RumbleInputView.axaml.cs:36]:
```suggestion
            RumbleInputView content = new(viewmodel);
```

File: src/Ryujinx.Ava/UI/Views/Input/MotionInputView.axaml.cs [hunk 4]
```diff
@@ -36,12 +35,12 @@ public MotionSettingsWindow(ControllerSettingsViewModel viewmodel)
             };
 
             InitializeComponent();
-            DataContext = _viewmodel;
+            DataContext = ViewModel;
         }
 
-        public static async Task Show(ControllerSettingsViewModel viewmodel)
+        public static async Task Show(ControllerInputViewModel viewmodel)
```
Comment (AcK77) [inline src/Ryujinx.Ava/UI/Views/Input/MotionInputView.axaml.cs:41]:
```suggestion
        public static async Task Show(ControllerInputViewModel viewModel)
```

File: src/Ryujinx.Ava/UI/Views/Input/MotionInputView.axaml.cs [hunk 5]
```diff
@@ -6,23 +6,22 @@
...
 
-        public MotionSettingsWindow()
+        public MotionInputView()
         {
             InitializeComponent();
-            DataContext = _viewmodel;
         }
 
-        public MotionSettingsWindow(ControllerSettingsViewModel viewmodel)
+        public MotionInputView(ControllerInputViewModel viewmodel)
```
Comment (AcK77) [inline src/Ryujinx.Ava/UI/Views/Input/MotionInputView.axaml.cs:20]:
```suggestion
        public MotionInputView(ControllerInputViewModel viewModel)
```

File: src/Ryujinx.Ava/UI/Views/Input/RumbleInputView.axaml.cs [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+        {
+            var config = viewModel.Configuration as InputConfiguration<GamepadInputId, StickInputId>;
+
+            _viewModel = new RumbleInputViewModel
+            {
+                StrongRumble = config.StrongRumble,
+                WeakRumble = config.WeakRumble
+            };
+
+            InitializeComponent();
```
Comment (AcK77) [inline src/Ryujinx.Ava/UI/Views/Input/RumbleInputView.axaml.cs:30]:
```suggestion
            InitializeComponent();
            
```

**Review by AcK77 — APPROVED** [review]
Just a missing comment, instead of that it LGTM!

**Review by TSRBerry — APPROVED** [review]
Lgtm!

**Pull request merged**